client/app/bundles/course/survey/pages/SurveyIndex/SurveysTable.jsx

```diff
@@ -115,10 +114,10 @@ class SurveysTable extends React.Component {
                   { survey.allow_response_after_end ? survey.time_bonus_exp : '-' }
                 </TableRowColumn>
                 <TableRowColumn colSpan={5}>
-                  { intl.formatDate(survey.start_at, standardDateFormat) }
+                  { moment(survey.start_at).format('DD-MM-YYYY') }
```

Comment:
at some point we should standard this as well.  now some places using `DD-MM-YYYY`, and some other places uses `DD MM YY`